rasa/core/training/converters/story_markdown_to_yaml_converter.py

```diff
@@ -41,12 +41,12 @@ async def convert_and_write(cls, source_path: Path, output_path: Path) -> None:
 
         # check if source file is test stories file
         if MarkdownStoryReader.is_test_stories_file(source_path):
-            reader = MarkdownStoryReader(unfold_or_utterances=False, use_e2e=True)
+            reader = MarkdownStoryReader(unfold_utterances=False, use_e2e=True)
```

Comment:
why do we need `unfold_utterances` if we have `use_e2e`. That's the same thing, isn't it?

Reply:
Because when you read a test story that will not be written later (i.g. the actual `rasa test` command with `use_e2e=True`), you actually want to parse the entities 😄

Reply:
Sure. So why not always unfold them?

Reply:
because we want to convert the original user stories and keep the "OR" statements they've been using.

Reply:
Ok, I see now why `e2e=True` is different to `unfold_utterances` 👍